components/input/gamepad/gamepad_mappings_linux.go

```diff
@@ -241,4 +245,40 @@ var GamepadMappings = map[string]Mapping{
 			298: input.ButtonMenu,
 		},
 	},
+	// Stadia Controller
+	stadiaPrefix: {
+		Axes: map[evdev.AbsoluteType]input.Control{
+			0:  input.AbsoluteX,
+			1:  input.AbsoluteY,
+			10: input.AbsoluteZ,
+			2:  input.AbsoluteRX,
+			5:  input.AbsoluteRY,
+			9:  input.AbsoluteRZ,
+			16: input.AbsoluteHat0X,
+			17: input.AbsoluteHat0Y,
+		},
+		Buttons: map[evdev.KeyType]input.Control{
+			304: input.ButtonSouth,
+			305: input.ButtonEast,
+			307: input.ButtonWest,
+			308: input.ButtonNorth,
+			310: input.ButtonLT,
+			311: input.ButtonRT,
+			314: input.ButtonSelect,
+			315: input.ButtonStart,
+			316: input.ButtonMenu,
+			317: input.ButtonLThumb,
+			318: input.ButtonRThumb,
+		},
+	},
+}
+
+// GetGamepadMapping returns the mapping for a given model.
+func GetGamepadMapping(model string) (Mapping, bool) {
```

Comment:
this is repetitive naming. can you do `MappingForModel`?

Reply:
Done.